### DISCUSSION TITLE
Review by commit checkboxes

### DISCUSSION THREAD

**Discussion by aastein** [community/community#13035] [Pull Requests]
PR files have a "Viewed" checkbox which reviewers can use to keep track of files they have already reviewed
<img width="157" alt="image" src="https://user-images.githubusercontent.com/18291415/159082819-c590eb5e-948d-4a86-ba4f-30ab31f8e454.png">

However the changes in PRs occur by commit, code is written by commit, and code is often reviewed by commit, not by file. I would like a similar feature as above that helps reviewers review PRs by commit. Maybe a checkbox next to the commit or something else.

**Comment by fedemengo** [comment]
This would be extremely useful

**Comment by bartek** [comment]
Ah, someone else made this issue! Yes please. This would be nice. We often review PRs by commit (as the work is broken down and more easy to parse), and being able to have an experience closer to the PR Review one would be helpful.
